DFC.ServiceTaxonomy.ContentPickerPreview/Views/ContentPickerField-Preview.Edit.cshtml

```diff
@@ -20,58 +20,54 @@
 <script asp-name="vue-multiselect-wrapper-preview" asp-src="~/DFC.ServiceTaxonomy.ContentPickerPreview/Scripts/vue-multiselect-wrapper-preview.js" at="Foot" depends-on="vuejs, vue-multiselect, sortable, vuedraggable"></script>
 <style asp-name="vue-multiselect-wrapper" asp-src="~/OrchardCore.ContentFields/Styles/vue-multiselect-wrapper.min.css" debug-src="~/OrchardCore.ContentFields/Styles/vue-multiselect-wrapper.css" depends-on="vue-multiselect"></style>
 
-<div class="form-group">
```

Comment:
Removing the `form-group` makes the content picker preview jump on open and close.